app/great_battle.py

```diff
@@ -0,0 +1,16 @@
+from app.structuring_the_knight_personality import Knight
+
+
+class BattleOfTwoKnight:
+    @staticmethod
```

Comment:
In case you use it only as a function collection or a collection of static methods, think of writing separate functions and collecting them in separate files.

Reply:
👍